website/Makefile

```diff
@@ -94,3 +94,13 @@ local-preview-build: clean setup structured-data preview-build run-link-checker
 # Generate Lighthouse scores locally
 lighthouse-report:
 	yarn lighthouse
+
+# Hot reload dev environment for the website
+local-server:
+	hugo server \
+	--disableFastRender \
+	 -verbose \
+	 --debug
```

Comment:
I think the `make serve` target is intended to enable local development. Maybe the flags for that one need to be updated?

Reply:
yea totally missed that when i was looking through this list, i'll remove